server/models/Profile.js

```diff
@@ -11,13 +16,18 @@ const profileSchema = new mongoose.Schema({
     },
     phoneNumber: Number,
     address: String,
+    city: String,
     description: String,
     availability: [
         {
             start: Date,
             end: Date,
         }
```

Comment:
I know this isn't part of this ticket but just something I'm thinking about. We might want to discuss availability on Tuesday. Is it going to be a daily availability time? Or is it going to only be certain dates? The way it is set up right now you'd have to say I'm available May 23rd - June 2022 for example, which might not make a lot of sense. The end date should probably be indefinite.

Reply:
That's a very good point. I will have to give it some thought and let's chat about it on Tuesday. Thanks!